decidim-admin/app/views/decidim/admin/resource_permissions/_options_form.html.erb

```diff
@@ -1,9 +1,14 @@
 <%
   handler_name = local_assigns[:handler_name]
+  ephemeral_warning = local_assigns[:ephemeral_warning]
```

Comment:
why the extra assignment ?

Reply:
This argument is passed to the partial with the authorization method options and displays a warning message for all the ephemeral authorization methods. This message warns that if the configuration of ephemeral authorizations is changed the existing authorizations of the same type will be revoked.